pylot/planning/utils.py

```diff
@@ -173,9 +173,9 @@ def stop_for_agents(ego_vehicle_location,
                     logger.debug(
                         '@{}: vehicle {} reduced speed factor to {}'.format(
                             timestamp, obstacle, speed_factor_v))
-        else:
-            logger.debug('@{}: filtering obstacle {}'.format(
-                timestamp, obstacle))
+        # else:
```

Comment:
Can you please keep logging of filtered obstacles?

Reply:
Yup, sorry forgot to uncomment this part! I just noticed that it floods the debug logs.